src/sentry/models/integrationfeature.py

```diff
@@ -13,6 +13,7 @@ class Feature:
     INCIDENT_MANAGEMENT = 5
     FEATURE_FLAG = 6
     ALERTS = 7
+    DEPLOYMENT = 8
```

Comment:
I suppose we need the changes to this file so that we can add the features in the migration later?

Reply:
@manuzope yes, we will use it here: https://github.com/getsentry/getsentry/blob/8b202e8723fb7045178e033deddb813b455b3457/getsentry/migrations/0055_add_integration_features_komodor.py#L8-L9